pkg/configurator/client_test.go

```diff
@@ -255,7 +180,7 @@ func TestMeshConfigEventTriggers(t *testing.T) {
 				meshConfig.Spec.Observability.Tracing.Port = int16(port)
 			case envoyLogLevelKey:
 				meshConfig.Spec.Sidecar.LogLevel = mapVal
-			case enableDebugServer:
+			case enableDebugServerKey:
```

Comment:
deltaMeshConfigContents no longer needs to be a map with the keys, you can use the meshConfig object in the test case. Please update this test to remove reference to keys and map

Reply:
Updated test cases. Can you check again?